commons/src/main/java/com/afollestad/materialdialogs/color/ColorChooserDialog.java

```diff
@@ -569,16 +572,16 @@ private Builder getBuilder() {
     return (Builder) getArguments().getSerializable("builder");
   }
 
-  private void dismissIfNecessary(AppCompatActivity context, String tag) {
-    Fragment frag = context.getSupportFragmentManager().findFragmentByTag(tag);
+  private void dismissIfNecessary(FragmentManager fragmentManager, String tag) {
+    Fragment frag = fragmentManager.findFragmentByTag(tag);
     if (frag != null) {
       ((DialogFragment) frag).dismiss();
-      context.getSupportFragmentManager().beginTransaction().remove(frag).commit();
+      fragmentManager.beginTransaction().remove(frag).commit();
     }
   }
 
   @NonNull
-  public ColorChooserDialog show(AppCompatActivity context) {
+  public ColorChooserDialog show(FragmentManager fragmentManager) {
```

Comment:
In order to do that, the default `show()` method would have to assume the passed in `Context` is an `Activity`, and try to glean the `FragmentManager` from that (as it used to). This would mean that the `Dialog` would crash (or otherwise fail to show) if presented from a `Fragment` without using the overloaded `show()` method. I can see this being more confusing for someone trying to understand how to use these dialogs.

I kind of understand where you're coming from, but I think it's safer to force the caller to pass in the correct `FragmentManager`. And although it's a 'widely used' method, remember that this is only for these 3 types of custom dialogs, which are different to your normal `MaterialDialogs` (which do not require a `FragmentManager` at all)

I think it's important to allow these `Dialogs` to be hosted somewhere other than an `Activity`, and it's not safe to then assume by default that they are shown from an `Activity`.